cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/ProgramController.java

```diff
@@ -115,7 +121,9 @@ public ProgramRunStatus getRunStatus() {
 
   ListenableFuture<ProgramController> resume();
 
-  ListenableFuture<ProgramController> stop();
+  ListenableFuture<ProgramController> complete();
```

Comment:
Removed `complete` from interface and made it as a part of `AbstractProgramController`